src/renderer/Core/Search/SearchResultList.tsx

```diff
@@ -23,6 +24,11 @@ export const SearchResultList = ({
 }: SearchResultListProps) => {
     const { t } = useTranslation();
 
+    const { value: scrollBehaviour } = useSetting<ScrollBehavior>({
+        key: "window.scrollBehavior",
+        defaultValue: "smooth",
+    });
```

Comment:
Please use a consistent name for this variable. Use "scroll behavior" (American English) instead of "scroll behaviour" (british english).

```suggestion
    const { value: scrollBehavior } = useSetting<ScrollBehavior>({
        key: "window.scrollBehavior",
        defaultValue: "smooth",
    });
```